_posts/2024-02-20-announcing-trino-fest-2024.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Trino Fest goes to Boston in 2024"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: false
+---
+
+After the resounding success of Trino Fest and Trino Summit 2023, Commander Bun
+Bun has exciting news to share: we're taking our biggest events of the year back
+to being in-person. They'll be hybrid, to be more specific, so if you can't
+travel, don't fret, you'll still be able to tune in and ask questions in chat.
+But if you can travel, you won't want to miss out! Everything you already know
+and love about Trino Fest is moving to the East Coast for the lovely Boston
```

Comment:
?? 

```suggestion
and love about Trino Fest is moving to the East coast for the lovely Boston
```

Reply:
https://english.stackexchange.com/questions/242399/east-coast-east-coast-or-east-coast

Because I'm referring to it as "the East Coast," it should be capitalized.